Sources/BraveWallet/Crypto/AssetIconView.swift

```diff
@@ -160,7 +162,7 @@ struct NFTIconView: View {
   }
   
   var body: some View {
-    NFTImageView(urlString: url?.absoluteString ?? "") {
+    NFTImageView(urlString: url?.absoluteString ?? "", isLoading: isLoadingMetadata) {
       AssetIconView(
```

Comment:
I think we'd want to show this `LoadingNFTView` as the placeholder icon now too (shown while fetching image from metadata, no image url in metadata, or when failing to fetch image)?

Latest desktop Nightly seems to be using it already as the placeholder image ([discussion thread](https://bravesoftware.slack.com/archives/C023VS4HJ6Q/p1699302078449009)):
<img width="523" alt="Screenshot 2023-11-13 at 1 43 05 PM" src="https://github.com/brave/brave-ios/assets/5314553/f6001551-1870-4e9c-a381-80d9172443d6">

Reply:
updated existed NFT placeholder with the new loading view without shimmering [9e04ade](https://github.com/brave/brave-ios/pull/8382/commits/9e04adebc1fdc6a189ad9b623a666787a3b20109)